aws/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -22,7 +22,7 @@ variable "security_groups" {
 
 # instances
 
-variable "count" {
+variable "worker_count" {
```

Comment:
This is a breaking change for all the providers. It would be nice to list somewhere (PR description?) all the breaking changes needed to upgrade to 0.12 support (like this change :))? What do you think?

Reply:
I think that should be done in separate PR.